armi/reactor/reactorParameters.py

```diff
@@ -366,25 +399,25 @@ def defineCoreParameters():
 
         pb.defParam(
             "maxDetailedDpaThisCycle",
-            units="dpa",
+            units=units.DPA,
             description="Max increase in dpa this cycle (only defined at EOC)",
         )
 
         pb.defParam(
             "dpaFullWidthHalfMax",
-            units="cm",
+            units=units.CM,
             description="Full width at half max of the detailedDpa distribution",
         )
 
         pb.defParam(
             "elevationOfACLP3Cycles",
-            units="cm",
+            units=units.CM,
             description="minimum axial location of the ACLP for 3 cycles at peak dose",
         )
 
         pb.defParam(
             "elevationOfACLP7Cycles",
```

Comment:
This parameter is very reactor-specific and should probably be moved to a plugin eventually.

Reply:
My preferred approach to this would be to open a ticket in the downstream repo plugin you want to move it to.  Because (a) that is probably and internal repo, and (b) work will need to be done in this repo AND that one.